modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/price/CommerceOrderPriceCalculationV2Impl.java

```diff
@@ -210,6 +210,42 @@ public CommerceOrderPrice getCommerceOrderPrice(
 				shippingDiscountedWithTaxAmount =
 					shippingDiscountedWithTaxAmount.subtract(
 						discountAmountCommerceMoney.getPrice());
+
+				CommerceDiscountValue
+					orderShippingCommerceDiscountValueWithoutTax =
```

Comment:
@brianikim we should name this 
```suggestion
					orderShippingWithoutTaxCommerceDiscountValue =
```